### DISCUSSION TITLE
Is the Docker install supposed to include the storefront or do I install that separately?

### DISCUSSION THREAD

**Discussion by paulcarl** [saleor/saleor#13157] [Q&A]
I've got Docker installed locally and followed the directions to install Saleor. It only has the Dashboard and Playground and I've been Googling and trying everything to try to also install the storefront into the container so I can try to work on my idea.

I've been at it for about 3 hours and I'm stumped. Some YouTube videos made it seem like the storefront is included in the saleor-platform so I'm wondering if maybe there's some error that I'm not seeing when I install it? I've seen some solutions where the docker-compose YAML is edited but that hasn't worked for me.

**Comment by paulcarl** [comment]
Came back with fresh eyes and I now realize that the front-end is supposed to be included, yet it isn't installing it when I run the Docker install (have done this multiple times now). The only warning/error I've found during installation is this:

WARNING saleor.core.jwt_manager RSA_PRIVATE_KEY is missing. Using temporary key for local development with DEBUG mode. [PID:1:MainThread]

I'm assuming that this must be what's causing the storefront to not load at all. My container only has 7 things in it (sorry, don't know the proper term, but Docker shows 7/7 when its running). Googling the error gives results for people who are having trouble moving to production but I'm just trying to tinker on my local machine. No idea how to proceed from here. Will update again if I figure out more information just in case other idiots like me are out there trying to figure this out too lol.

**Reply by popaurel** [reply]
The [storefront is a separate GitHub project and comes with it's own docker-compose.yml](https://github.com/saleor/storefront) so that people can deploy it on a different server from the [backend (deployed via docker-compose.yml of saleor-platform)](https://github.com/saleor/saleor-platform), which is the main purpose of headless apps.
